Keep all upload data persistent and advice on redirections issues (#202)
martialblog/docker-limesurvey: README.md

See:
- https://github.com/martialblog/docker-limesurvey/issues/127

## Updating or rebuilding container removes themes/plugins/...

The `/var/www/html/upload` directory contains multiple subdirectories for different kind of persistent data,
e.g. themes, plugins and survey uploads. In past, the example configuration recommended only the persistent
volume for survey uploads. In such a situation, any rebuild of the container will remove other stored data.

> To keep them persisten, you need to ensure that all subdirectories of `upload` are stored in a volume.

Reviewer: ```suggestion
To keep them persistent, you need to ensure that all subdirectories of `upload` are stored in a volume.
```

The easiest way is to mount the whole `/var/www/html/upload`. If you want to update an existing deployment
from previous configuration, please keep in mind that just changing the mount path may cause data loss 
(e.g. already uploaded themes). Please backup your data from the container before any modifications.

> You may also consider separated volumes for every subdirectory.

Reviewer: ```suggestion
You may also consider separated volumes for each subdirectory.
```

## Apache generates redirects with internal port / URLs don't work without trailing slash

> If your deployment is behind additional proxy that listen on a different port than the Apache in the

Reviewer: ```suggestion
If your deployment is behind an additional proxy that listens on a different port than the Apache in the
```

container, you may notice some issues with redirects generated from Apache, e.g. when redirecting
URLs to the cannonical version. Apache will try to add the port it's listening on to the URL.

> To prevent this, you may edit the Apache configuration and add `ServerName` directive with your

Reviewer: ```suggestion
To prevent this, you may edit the Apache configuration and add the `ServerName` directive with your
```
